### PR TITLE
feat(provider) add MeliMedia movie provider

### PR DESCRIPTION
**Pull request by Teekzilla**
## feat(provider) add MeliMedia movie provider
Added MeliMedia as a new movie provider. Everything is pretty straight forward, I'll add the testing required tag for now as well as request a review. I also forgot to archive `GoStream` when I cleaned up providers so that's in this as well.

### What's in this PR:
- Added MeliMedia as a new universal provider.
- Moved GoStream movie provider to archive.

### CODE REVIEW

File: src/scrapers/providers/movies/MeliMedia.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            let searchUrl = this._generateUrl(url, { s: searchTitle });
+            const rp = this._getRequest(req, ws);
+            const jar = rp.jar();
+            const response = await this._createRequest(rp, searchUrl, jar, headers);
+
+            let $ = cheerio.load(response);
+
+            let videoPage = '';
+            $('.-title a').toArray().forEach(element => {
+                // Replace is used because Farda uses a different apostrophe character...
```
Comment (ddivad) [inline src/scrapers/providers/movies/MeliMedia.js]: I guess this comment should be updated, not Farda ;) The reason I don't like comments haha

File: src/scrapers/resolvers/MeliMedia.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const rp = require('request-promise');
```
Comment (ddivad) [inline src/scrapers/resolvers/MeliMedia.js]: I guess these imports aren't needed here?

File: src/scrapers/providers/movies/MeliMedia.js (hunk 1 above)
Comment (Teekzilla) [reply src/scrapers/providers/movies/MeliMedia.js]: Updated.

File: src/scrapers/resolvers/MeliMedia.js (hunk 2 above)
Comment (Teekzilla) [reply src/scrapers/resolvers/MeliMedia.js]: Yup, I'll go fix up other imports as well separately as I think there may be a lot of unused imports throughout.

**Comment by Teekzilla** [comment]
This closes #100.

**Pull request merged**